Sources/WaterfallGrid/GridSyle.swift

```diff
@@ -21,7 +21,7 @@ struct GridSyle {
         #elseif os(watchOS)
         return columnsInPortrait
         #else
-        return UIDevice.current.orientation.isLandscape ? columnsInLandscape : columnsInPortrait
+        return UIScreen.main.bounds.size.width > UIScreen.main.bounds.size.height ? columnsInLandscape : columnsInPortrait
```

Comment:
thanks, this looks good to me!

Can I just please ask to break this line, to reduce its length, in something like this:

```swift
let screenSize = UIScreen.main.bounds.size
return screenSize.width > screenSize.height ? columnsInLandscape : columnsInPortrait
```